### PR TITLE
Bundle CSS changes

### PR DESCRIPTION
**Pull request by nabeelio**
@bbuske How's this look?

### CODE REVIEW

File: resources/views/layouts/default/flights/nav.blade.php [hunk 1]
```diff
@@ -1,27 +1,7 @@
 <div class="card">
     <div class="card-block" style="min-height: 0px; display: flex; justify-content: center; align-items: center;">
-        <style>
```
Comment (nabeelio) [inline resources/views/layouts/default/flights/nav.blade.php]: Moved to `public/assets/frontend/css/styles.css`

File: resources/views/layouts/default/flights/search.blade.php [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
 <div class="card pull-right">
     <div class="card-block" style="min-height: 0px">
-        <div class="form-group">
+        <div class="form-group search-form">
             {{ Form::open([
                     'route' => 'frontend.flights.search',
                     'method' => 'GET',
                     'class'=>'form-inline'
             ]) }}
-            <style>
```
Comment (nabeelio) [inline resources/views/layouts/default/flights/search.blade.php]: Moved to `public/assets/frontend/css/styles.css`

**Comment by bbuske** [comment]
@nabeelio Looks goos. I have not yet pushed the new update on the server. I am a bit busy this week, flying... I will push ASAP and let you know.

**Comment by nabeelio** [comment]
No problem, thanks! If there's a problem we can address it

**Comment by nabeelio** [comment]
Sounds good! Just another fresh install on dev making sure it's working still. I'll probably cut the next beta soon barring anything major. I know there's a few small things you brought up which I'm going to try to get to today/tomorrow. But then I'm also out until Sunday.

**Comment by bbuske** [comment]
No worries @nabeelio. 
No rush... :)

**Pull request merged**